xpdacq/xpdacq_conf.py

```diff
@@ -180,14 +184,47 @@ def configure_frame_acq_time(new_frame_acq_time):
     print("INFO: area detector has been configured to new "
           "exposure_time = {}s".format(new_frame_acq_time))
 
-def _load_beamline_config(beamline_config_fp):
+def _verify_within_test(beamline_config_fp,verif):
+    while verif != "y":
```

Comment:
Is there a reason to not do this with an `if` statement?

Reply:
I just tried to leave it as close to the non-test syntax as possible, but can change it if you prefer.

Reply:
No that's a fair point, its fine as is.